qw5q_platinum/platform.py

```diff
@@ -0,0 +1,77 @@
+import pathlib
+
+from qibolab import (
+    AcquisitionChannel,
+    Channel,
+    ConfigKinds,
+    DcChannel,
+    IqChannel,
+    Platform,
+    Qubit,
+)
+from qibolab.instruments.qm import Octave, QmConfigs, QmController
+from qibolab.instruments.rohde_schwarz import SGS100A
+
+FOLDER = pathlib.Path(__file__).parent
+
+# Register QM-specific configurations for parameters loading
+ConfigKinds.extend([QmConfigs])
+
+
+def create():
+    """Line D of QuantWare 21q-chip controlled with Quantum Machines.
+
+    Qubits D1, D2, D3 have been tested to work.
+    """
```

Comment:
I guess this is a leftover from a copy...

Reply:
Fixed in 49467d3e1f02a5a377990fea6b8321ba3be8879c, thanks.